src/main/java/com/ericsson/ei/queryservice/ProcessAggregatedObject.java

```diff
@@ -42,38 +46,67 @@
     @Value("${database.name}")
     private String aggregationDataBaseName;
 
-    static Logger log = (Logger) LoggerFactory.getLogger(ProcessAggregatedObject.class);
+    private static final Logger LOGGER = (Logger) LoggerFactory.getLogger(ProcessAggregatedObject.class);
 
     @Autowired
-    MongoDBHandler handler;
+    private MongoDBHandler handler;
 
     /**
      * The method is responsible to extract the aggregated data on the basis of
      * the ID from the aggregatedObject.
-     * 
+     *
      * @param id
      * @return ArrayList
      */
     public ArrayList<String> processQueryAggregatedObject(String id) {
         ObjectMapper mapper = new ObjectMapper();
         String condition = "{\"_id\" : \"" + id + "\"}";
-        log.info("The condition is : " + condition);
+        LOGGER.debug("The condition is : " + condition);
         JsonNode jsonCondition = null;
         try {
             jsonCondition = mapper.readTree(condition);
         } catch (Exception e) {
-            log.error(e.getMessage(), e);
+            LOGGER.error(e.getMessage(), e);
         }
-        log.info("The Json condition is : " + jsonCondition);
+        LOGGER.debug("The Json condition is : " + jsonCondition);
         ArrayList<String> response = handler.find(aggregationDataBaseName, aggregationCollectionName,
                 jsonCondition.toString());
         return response;
     }
 
+    /**
+     * This method is responsible for fetching all the aggregatedObjects from
+     * the Aggregation database and return it as JSONArray.
+     *
+     * @param request
+     * @param AggregationDataBaseName
+     * @param AggregationCollectionName
+     * @return JSONArray
+     */
+    public JSONArray processQueryAggregatedObject(JsonNode request, String AggregationDataBaseName, String AggregationCollectionName) {
+        DB db = new MongoClient().getDB(AggregationDataBaseName);
```

Comment:
please remove deprecated method usage, 
Check the below link, we removed deprecated  .getDB and collections
https://github.com/Ericsson/eiffel-intelligence/blob/master/src/main/java/com/ericsson/ei/mongodbhandler/MongoDBHandler.java

Reply:
I can't avoid this, because Jongo use only DB

Reply:
Yes, Issue is going on this, 
https://github.com/bguerout/jongo/issues/254